Lib/uuid.py

```diff
@@ -601,9 +638,19 @@ def _windll_getnode():
         return UUID(bytes=bytes_(_buffer.raw)).node
 
 def _random_getnode():
-    """Get a random node ID, with eighth bit set as suggested by RFC 4122."""
+    """Get a random node ID."""
+    # RFC 4122, $4.1.6 says "For systems with no IEEE address, a randomly or
+    # pseudo-randomly generated value may be used; see Section 4.5.  The
+    # multicast bit must be set in such addresses, in order that they will
+    # never conflict with addresses obtained from network cards."
```

Comment:
I prefer "the multicast bit" rather than "eight bit". Please update also the documentation:
https://docs.python.org/dev/library/uuid.html#uuid.getnode